framework/flamingo/sessions.go

```diff
@@ -55,6 +57,7 @@ func (m *SessionModule) Inject(config *struct {
 	Path                 string  `inject:"config:flamingo.session.cookie.path"`
 	RedisURL             string  `inject:"config:flamingo.session.redis.url"`
 	RedisHost            string  `inject:"config:flamingo.session.redis.host"`
+	RedisKeyPrefix       string  `inject:"config:flamingo.session.redis.keyPrefix,optional"`
```

Comment:
Could you add the key to the `CueConfig` method for validation and also document it in the `SessionReadme.md` file?

Reply:
Hello Karsten, 
thx for your quick feedback. I updated ReadmeSession.md and added an entry in CueConfig(). Is there any reason, why the `?` for indicating optional parameters wasn't used so far?

Reply:
@karstenkoehler Regarding `CueConfig()` there is a second option: `keyPrefix: string | ""`. Please let me know, if you prefer the other one.